src/main/resources/db/changelog/changes/v4.0/add-additional-fields-to-authority-archive.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                   http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.18.xsd">
+
+  <changeSet id="add-additional-fields-to-authority-archive" author="Elena_Shmygaliova">
+    <preConditions>
+      <tableExists tableName="authority_archive"/>
+    </preConditions>
+
+    <comment>Add new jsonb fields to authority_archive table</comment>
+
+    <addColumn tableName="authority_archive">
+      <column name="additional_headings" type="jsonb"/>
+    </addColumn>
```

Comment:
Why it was decided to have only one jsonb field?

Reply:
just to minimize number of new fields/columns in database

Reply:
What benefits will we gain from this approach? I want to ensure that it won't require us to undertake refactoring in the future.

Reply:
If we want to add other additional fields in future we don't have to add new columns to the table in db, we just need to extend a single "additinal_fields" jsonb column. It required minor change comparing to column-per-fieldType approach.